src/rpc/client.fake.ts

```diff
@@ -13,12 +13,10 @@ const ServerRuntime = ManagedRuntime.make(ServerLayer);
 
 class ServerClient extends Effect.Service<ServerClient>()("ServerClient", {
   scoped: Effect.gen(function* () {
-    const rpc = yield* RpcTest.makeClient(Procedures).pipe(
-      Effect.provide(ProceduresLive),
```

Comment:
Effect.provide should be avoided here, as it will shutdown the layer once the effect finishes (which is immediately here)